src/assets/scripts/client/ui/UiController.js

```diff
@@ -662,6 +677,24 @@ class UiController {
         this.$switchAirport.addClass(SELECTORS.CLASSNAMES.ACTIVE);
     }
 
+
+    /**
+     * @for UiController
+     * @method onInitiateAirportSearch
+     */
+
+    onInitiateAirportSearch() {
+        EventTracker.recordEvent(TRACKABLE_EVENT.AIRPORTS, 'airport-search', 'start');
+
+        const value = this.$airportDialog.find(SELECTORS.DOM_SELECTORS.AIRPORT_SEARCH).val().toLowerCase();
+
+        $('.dialog-body li').each(
```

Comment:
This isn't the correct place for this, as you guessed, but you're close.  I'll follow up with some more info on where this should actually live.

Reply:
Thank you!

Reply:
@n8rzz do you have a suggestion as to where this should be moved?